identifiers/os_product.txt

```diff
@@ -116,7 +116,6 @@ IPReach
 IPSO
 IRIX
 Integrated Lights Out Manager
-Integrated Lights Out Manager firmware
```

Comment:
I rebuilt all of the `identifiers/` files. Apparently this value was no longer used.